src/coreclr/src/ToolBox/superpmi/superpmi-shared/compileresult.cpp

```diff
@@ -796,6 +796,16 @@ void CompileResult::applyRelocs(unsigned char* block1, ULONG blocksize1, void* o
                 size_t address = section_begin + (size_t)fixupLocation - (size_t)originalAddr;
                 if ((section_begin <= address) && (address < section_end)) // A reloc for our section?
                 {
+#if defined(TARGET_AMD64)
```

Comment:
do we need it for x86?

Reply:
No; on x86 these don't need relocation because the 32-bit immediate can handle any offset.